frontend/src/pages/IntegrationsPage/components/GitHubIntegration/GitHubIntegrationConfig.tsx

```diff
@@ -34,7 +34,9 @@ const GitHubIntegrationConfig: React.FC<
 			<>
 				<p className={styles.modalSubTitle}>
 					Disconnecting your GitHub workspace from Highlight will
-					prevent you from linking issues to future comments
+					prevent you from linking issues to future comments and your
+					backend errors will no longer be enhanced with stacktraces
+					from GitHub.
```

Comment:
`and your backend errors will no longer be enhanced with stacktraces from GitHub.` -> `and your stacktraces will not be enhanced`